models/model.js

```diff
@@ -56,3 +56,17 @@ exports.fetchArticlesSortedByDate = (order) => {
     return article.rows;
   });
 };
+
+exports.fetchArticleComments = (article_id) => {
+  if (article_id > 36) {
```

Comment:
Interesting work around, but what happens when more articles are added. You'll want to explore a more secure way of handling both scenarios. 
Maybe you could make a request for that article_id, to check it exists? 
You might want to take a look at the Promise.all to help with this.